<launch>
    <node name="topo_send_image" pkg="topo_maps" type="topo_send_image">
        <param name="unknown_min" value="127" type="int"/>
        <param name="unknown_max" value="128" type="int"/>
        <param name="resolution" value="0.1" type="double"/>
        <param name="image_file" type="string"
            value="$(find topo_maps)/nrl-evg/sandbox/maps/strongly_connected.pgm"/>
        <param name="fixed_frame" type="string" value="map"/>
        <param name="occupancy_topic" type="string"
            value="topo_occupancy_grid"/>
    </node>
</launch>
